src/routes/articles.rs

```diff
@@ -38,61 +36,79 @@ pub fn post_articles(
     let body = extractor.extract("body", new_article.body);
     extractor.check()?;
 
-    let article = db::articles::create(
-        &conn,
-        auth.id,
-        &title,
-        &description,
-        &body,
-        &new_article.tag_list,
-    );
+    let article = db
```

Comment:
This is an example of why I renamed the `db` module to `database`.
To differentiate between the module and the wrapped connection coming into the function.